### PR TITLE
DOCS: Removing #archipelago-dev from places

### PR DESCRIPTION
**Pull request by ReverM**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

DOCS: Changing #archipelago-dev to #ap-world-dev

## What is this fixing or adding?

See above

## How was this tested?

I read

### CODE REVIEW

**Review by ScipioWright — COMMENTED** [review]
idk if it should be ap-world-dev or ap-core-dev, but the core review can figure that out

**Review by Exempt-Medic — APPROVED** [review]
LGTM. These are the only instances and they are changed to the proper of the two channels

**Comment by ReverM** [comment]
> idk if it should be ap-world-dev or ap-core-dev, but the core review can figure that out

Yeah fair. I think it makes more sense in ap-world-dev tho?

**Review by nicholassaylor — APPROVED** [review]
LGTM

File: docs/world maintainer.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ When a world is unmaintained, the [core maintainers](https://github.com/orgs/Arc
 can vote for a new maintainer if there is a candidate.
 For a vote to pass, the majority of participating core maintainers must vote in the affirmative.
 The time limit is 1 week, but can end early if the majority is reached earlier.
-Voting shall be conducted on Discord in #archipelago-dev.
+Voting shall be conducted on Discord in #ap-world-dev.
```
Comment (black-sliver) [inline docs/world maintainer.md:38]: I'm not sure if this should be core instead 🤔 opinions?

Comment (NewSoupVi) [reply docs/world maintainer.md:38]: Not a strong one

I do see your point about it being a core thing so it should be in ⁠ap-core-dev 

But then other other hand, it's something where the discussion would probably start in ⁠ap-world-dev, so we'd have to "move over" at some point for the poll I guess?

Comment (black-sliver) [reply docs/world maintainer.md:38]: No idea, I'd be fine with either, however I think previous communications of that nature were done in a thread that was not visible to the public?

Comment (black-sliver) [inline docs/world maintainer.md:38]:
```suggestion
Voting shall be conducted on Discord in #ap-core-dev.
```

File: docs/world maintainer.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ for example when they become unreachable.
 For a vote to pass, the majority of participating core maintainers must vote in the affirmative.
 The time limit is 2 weeks, but can end early if the majority is reached earlier AND the world maintainer was pinged and
 made their case or was pinged and has been unreachable for more than 2 weeks already.
-Voting shall be conducted on Discord in #archipelago-dev. Commits that are a direct result of the voting shall include
+Voting shall be conducted on Discord in #ap-world-dev. Commits that are a direct result of the voting shall include
```
Comment (black-sliver) [inline docs/world maintainer.md:54]:
```suggestion
Voting shall be conducted on Discord in #ap-core-dev. Commits that are a direct result of the voting shall include
```

File: docs/world maintainer.md (hunk 1 above)
Comment (NewSoupVi) [reply docs/world maintainer.md:38]: That's also true haha

**Review by black-sliver — CHANGES_REQUESTED** [review]
I'm gonna make the call. Since this is about inclusion of a world in core rather than about world development, I believe -core is the right place and the archived previous threads are also there.

**Comment by ReverM** [comment]
Changed have been made

**Review by black-sliver — APPROVED** [review]
Thanks!

**Pull request merged**